### ISSUE TITLE
Security Warning, not redirecting...

### ISSUE THREAD

**Issue by h4h4** [ccoenraets/OpenFB#71]
Hi, I know many people had already posted this issue, and I tried those solutions. 
It worked very well on browser and emulate. but it stocked me for days that I can't use it on my ios device. 
still seeing this after login and not redirecting to any page.. : (

Success
SECURITY WARNING: Please treat the URL above as you would your password and do not share it with anyone.

my Valid OAuth redirect URIs are as instruction
(http://localhost:8100/oauthcallback.html,https://www.facebook.com/connect/login_success.html)

, and my plugins version as below. 

0.11.0 "Facebook Connect"
1.0.0 "InAppBrowser"
1.0.0 "Whitelist"

I saw someone end up with order problem, so add it here if necessary.

<script src="lib/ionic/js/ionic.bundle.js"></script>

<script src="lib/ngCordova/dist/ng-cordova.js"></script>

<script src="cordova.js"></script>

<script src="js/openfb.js"></script>

<script src="js/ngopenfb.js"></script>

<script src="js/app.js"></script>

...

and I'm using the "Ionic View App" to test my App. ( Will that be a problem? )

Any advice would be really appreciate, thanks all in advance.

**Comment by sajclarke** [comment]
Hi, can you check your facebook app to ensure you have added the "website" platform? 

In Facebook Developers Console, go to your app then go to "Basic" section and under the section with App ID, click "Add Platform" and choose "website". You do not have to add any of the URLs since you're using cordova. 

Also make sure that you have turned on OAuth Login in the "Advanced" section

This worked for me. Let me know if it works for you

**Comment by h4h4** [comment]
Hi, sorry for the late reply.
I just tried to add the website in Basic tab, but facebook said "This or Mobile Site URL is required."
so I can't leave it blank, and my OAuth Login is turned on already.
I saw someone place localhost:8888 and worked, but it's not work for me, too.
Did I missed anything?

I tried my app with my friend's android yesterday , it worked pretty well !
I found that the eventListener 'loadstart and 'exit' is not even entering in ios.('runningInCordova' is true)

I mean 

if (runningInCordova) {  
            loginWindow.addEventListener('loadstart', loginWindow_loadStartHandler);
            loginWindow.addEventListener('exit', loginWindow_exitHandler);
}

the eventlistener added successfully.  but not doing anything when load start.

any suggests?
